packages/jest-config/src/__tests__/normalize.test.js

```diff
@@ -684,7 +684,9 @@ describe('babel-jest', () => {
   beforeEach(() => {
     Resolver = require('jest-resolve');
     Resolver.findNodeModule = jest.fn(
-      name => path.sep + 'node_modules' + path.sep + name,
+      name => {
```

Comment:
can you remove the curly brace + return again? Doesn't seem necessary here and Jest's code style is to not use these when it's a single expression.